### PR TITLE
New changelog 2nd of sept

### PR DESCRIPTION
**Pull request by springcoil**
Changelog update with new voice builder, new script improvements and new connectors

### CODE REVIEW

**Review by timcoggins — APPROVED** [review]
Looks good to me

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,79 @@
...
+* simplicity of version-specific changes (e.g. SDK-side transformations, API-side version-specific feature flags, extra keys ignored in previous versions etc.) 
+
+### New voices
+* We now have over 600 voices! Improving our customer experience.
+* We introduced a new partner [DeepZen](https://www.aflorithmic.ai/post/welcoming-new-deepzen-voices-into-api-audio-by-aflorithmic) which have amazing emotive voices. Give them a try :) 
+* You can listen to them [here in the library](https://library.api.audio/voices?providerFullName=deepzen)
+
+### Better voices
+We recently migrated our ML platform from an old platform to a new platform. (Older version of kubeflow to newer version of kubeflow)
+We've rolled the out to all customers.
```
Comment (Li-create2021) [inline CHANGELOG.md:16]: This sentence doesn't make sense?

**Review by Li-create2021 — CHANGES_REQUESTED** [review]
Line 16 doesn't make sense

File: CHANGELOG.md (hunk 1 above)
Comment (springcoil) [reply CHANGELOG.md:16]: Good point :P

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,79 @@
 # Changelog
+## Friday 9th September 2022
+### Voice Cloner SDK
+* [https://github.com/aflorithmic/apiaudio-python/pull/121](Voice Cloner SDK) - 
+* SDK's readability and the accuracy of API's representations (object members annotations in IDEs!),
```
Comment (GetOnMyLvl97) [inline CHANGELOG.md:5]: <3
Next SDK V2 with Classes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,79 @@
...
+* testability (proper client tests, increased probability of catching breaking changes),
+* simplicity of version-specific changes (e.g. SDK-side transformations, API-side version-specific feature flags, extra keys ignored in previous versions etc.) 
+
+### New voices
+* We now have over 600 voices! Improving our customer experience.
+* We introduced a new partner [DeepZen](https://www.aflorithmic.ai/post/welcoming-new-deepzen-voices-into-api-audio-by-aflorithmic) which have amazing emotive voices. Give them a try :) 
+* You can listen to them [here in the library](https://library.api.audio/voices?providerFullName=deepzen)
+
+### Better voices
+We recently migrated our ML platform from an old platform to a new platform. (Older version of kubeflow to newer version of kubeflow)
```
Comment (GetOnMyLvl97) [inline CHANGELOG.md:15]: I think we shouldn't need to tell anyone we're running kubeflow

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,79 @@
...
+* Fixed our security vulnerabilities - because we care about customer trust (this also addresses Log4J issues [we had](https://logging.apache.org/log4j/2.x/security.html))
+* New inference API, faster inference - faster voices
+* We’re able to roll out faster, more reliable, and do more frequent updates. Enabling us to develop better features, faster
+* The deployment is way easier (we created a python package) 
+* Our ML team is able to access logs from the dashboard, so able to fix bugs faster :)
+
+
+## Friday 2nd September 2022
+## Voice builder
+We improved our voice creation capabilities so we did automatic voice preview. We have automatic preview of files, as soon as the model finished training, and proper status updates. This enhances the customer experience in our ability to deliver great voices.
```
Comment (GetOnMyLvl97) [inline CHANGELOG.md:30]: zero shot is a surprise?

**Review by GetOnMyLvl97 — APPROVED** [review]
some comments

File: CHANGELOG.md (hunk 2 above)
Comment (martinezpl) [reply CHANGELOG.md:5]: That's about to come it's not rolled out. There should be a note here like what's cooking or coming soon or whatever but it's not a change that's been introduced yet.

And SDK v2 will come even later than that.

**Pull request merged**